Project.toml

```diff
@@ -81,19 +81,19 @@ RecipesBase = "1.1"
 Reexport = "1.0"
 Requires = "1.1"
 SciMLBase = "1.90, 2"
-Setfield = "0.8, 1"
+Setfield = "1"
 SimpleUnPack = "1.1"
 SparseArrays = "1"
-StartUpDG = "0.17"
-Static = "0.3, 0.4, 0.5, 0.6, 0.7, 0.8"
+StartUpDG = "0.17.7"
+Static = "0.8.7"
```

Comment:
De-supporting older versions of `Static` was intentional?

Reply:
Yes, we rely on features/bug fixes from Static.jl v0.8.7 (https://github.com/SciML/Static.jl/pull/110 to be precise).